Extensions/ArtifactEngine/xliff/de-DE.xlf

```diff
@@ -5,47 +5,47 @@
       <trans-unit id="loc.messages.DownloadSummary">
         <source>Total Files: %s, Processed: %s, Skipped: %s, Failed: %s, Download time: %s secs, Download size: %s</source>
         <target state="translated">Dateien gesamt: %s, verarbeitet: %s, übersprungen: %s, fehlerhaft: %s, Downloadzeit: %s Sek., Downloadgröße: %s</target>
-        <note />
+        <note/>
       </trans-unit>
       <trans-unit id="loc.messages.DownloadedTo">
         <source>Downloaded %s to %s</source>
         <target state="translated">Download von "%s" nach "%s" wurde durchgeführt.</target>
-        <note />
+        <note/>
       </trans-unit>
       <trans-unit id="loc.messages.DownloadingTo">
         <source>Downloading %s to %s</source>
         <target state="translated">Download von "%s" nach "%s" wird durchgeführt.</target>
-        <note />
+        <note/>
+      </trans-unit>
+      <trans-unit id="loc.messages.FailedToParseResponse">
```

Comment:
notice the case difference, FailedToParseResponse and failedToParseResponse. Its in sorted order, hence moved places..